starter-kits/product/_access.md

```diff
@@ -1,3 +1,4 @@
+% - at least one of access and/or use constraints required
```

Comment:
I am thinking that the "-" bullet points may confuse people because it is similar to the markdown syntax used in this product kit. Instead, maybe we could indicate that this is a comment by using something like:
```
% Note: At least one of access and/or use constraints required
```

Then we could do the same thing throughout, depending on what we decide.

```
% Note: Required content
```

Reply:
Good point. Removed.